app/src/main/java/fr/free/nrw/commons/category/CategoryContentProvider.java

```diff
@@ -42,12 +44,12 @@ public static Uri uriForId(int id) {
         return Uri.parse(BASE_URI.toString() + "/" + id);
     }
 
-    @Inject DBOpenHelper dbOpenHelper;
+    @Inject Lazy<DBOpenHelper> dbOpenHelper;
```

Comment:
A rename would make it clearer that this is a _provider_ of `DBOpenHelper` instances rather than the actual `DBOpenHelper`

Reply:
I realise that lazy injection is no longer needed. I tried using it while keeping `AndroidInjection` in place.